cleansio/censor/censor_file.py

```diff
@@ -22,13 +22,13 @@ def censor(self):
         clean_file = AudioSegment.empty()
         audio_file = AudioFile(self.file_path)
         # Define the CLI progress bar
-        p_bar, p_bar_step = self.__progress_bar(audio_file.chunks_file_paths)
+        p_bar, p_bar_step = self.__progress_bar(audio_file.normal_chunks)
         async_iter = zip(
             repeat(p_bar),
             repeat(p_bar_step),
-            audio_file.chunks_file_paths)
+            audio_file.normal_chunks)
         # Censor each audio chunk file asynchronously
-        censored_chunks = ThreadPool(8).map(self.__censor_chunk, async_iter)
+        censored_chunks = ThreadPool(6).map(self.__censor_chunk, async_iter)
```

Comment:
Changed to 6 because with overlapping chunks the number of GCS requests is multiplied by 2. 16 requests went past the limit, 12 seems fine.